eventRequirement.js

```diff
@@ -449,6 +449,30 @@ OperandVariable.prototype.getValue = function (parameters) {
                 return left % right;
             }
             return null;
+        case "arrayvalue":
+            var left = value.left.getValue(parameters);
+            var right = value.right.getValue(parameters);
+
+            if (left != null) {
+                if (left.hasOwnProperty('parentVar')) {
+                    left = left.toJS();
+                }
+            }
+            if (right != null) {
+                if (right.hasOwnProperty('parentVar')) {
+                    right = right.toJS();
+                }
+
+            }
+
+            if ($.isNumeric(right)) {
+                right = parseFloat(right);
```

Comment:
the right value should be an index, so it would be better to use parseInt instead of parseFloat.

Reply:
done